engine/src/main/java/org/terasology/world/block/family/FreeFormSupported.java

```diff
@@ -20,6 +20,10 @@
 import java.lang.annotation.RetentionPolicy;
 import java.lang.annotation.Target;
 
+/**
+ * Annotation used to mark if a {@link BlockFamily} supports blocks in freeform shapes.
```

Comment:
Annotation used to mark if a {@link BlockFamily} supports freeform shapes or blocks that have freeform shapes.